src/oracle/oracle.cairo

```diff
@@ -1746,29 +1746,30 @@ mod Oracle {
             let g_entry: PossibleEntries = IOracleABI::get_data_entry(self, data_type, source);
             match g_entry {
                 PossibleEntries::Spot(spot_entry) => {
-                    let is_entry_not_initialized: bool = spot_entry.get_base_timestamp() == 0;
-                    let condition: bool = is_entry_not_initialized
-                        && (spot_entry
-                            .get_base_timestamp() < (latest_timestamp - BACKWARD_TIMESTAMP_BUFFER));
-                    if !condition {
+                    let is_entry_initialized: bool = spot_entry.get_base_timestamp() != 0;
+                    let condition: bool = is_entry_initialized
+                        & (spot_entry
```

Comment:
why remove the `&&` condition ? Here we are performing a bit and operation.

Reply:
there are no clear syntax explicited by cairo, both works, we just need to stick to a version for clear code.